api/controllers/AuthController.js

```diff
@@ -86,6 +86,12 @@ module.exports = {
       inviteToken : req.param('inviteToken')
     };
 
+    // checking password for white spaces and tabs
```

Comment:
Not sure why you'd want to deviate from the standard. `\s` covers both whitespace and tab